addons/mail/static/src/core/common/discuss.scss

```diff
@@ -1,3 +1,8 @@
+// Substract the height of the ControlPanel component if it is displayed.
+.o_control_panel + .o-mail-Discuss {
+    max-height: calc(100% - 58px);
+}
```

Comment:
In stable I would not make change related to control panel, at least not in this PR. Not confident enough, at least this way we can progress with viewing livechat history.